docs/upgrading.md

```diff
@@ -104,6 +104,10 @@ To disable this set `metricsConfig.secure` to `false`.
 When returning a map or array in an expression, you would get a Golang representation.
 This now returns plain JSON.
 
+### Container name in error messages
```

Comment:
```suggestion
### Added container name to workflow node error messages
```

- this could be more specific
- it should also be in active voice of what was changed